jupyterhub/spawner.py

```diff
@@ -255,6 +320,17 @@ def get_env(self):
                 env[key] = value
 
         env['JPY_API_TOKEN'] = self.api_token
+
+        # Put in limit and guarantee info if they exist.
```

Comment:
Maybe not have it in the env here, because the subprocess doesn't understand it. Let's leave it to Spawners to take them from attributes and put them in the right place.